src/test/java/bio/terra/integration/IngestTest.java

```diff
@@ -346,62 +346,39 @@ public void ingestWildcardMalformedTest() throws Exception {
     IngestRequestModel mergeIngestRequest =
         dataRepoFixtures
             .buildSimpleIngest("sample", "ingest-test/merge/ingest-test-sample-merge.json")
             .updateStrategy(UpdateStrategyEnum.MERGE);
     IngestResponseModel mergeIngestResponse =
         dataRepoFixtures.ingestJsonData(steward(), datasetId, mergeIngestRequest);
     assertThat("correct merge sample row count", mergeIngestResponse.getRowCount(), equalTo(2L));
-
-    // Rows ingested via merge should not increase the existing live row count.
-    // TODO: once the preview API GA and works for datasets, we should use that here
-    DatasetModel dataset =
-        dataRepoFixtures.getDataset(
-            steward(), datasetId, List.of(DatasetRequestAccessIncludeModel.ACCESS_INFORMATION));
-    BigQueryProject bigQueryProject =
-        BigQueryProject.get(dataset.getAccessInformation().getBigQuery().getProjectId());
-    AccessInfoBigQueryModelTable bqTableInfo =
-        dataset.getAccessInformation().getBigQuery().getTables().stream()
-            .filter(t -> t.getName().equals("sample"))
-            .findFirst()
-            .orElseThrow();
-    // Note: the sample query is just a formatted select * query against the table
-    TableResult bqQueryResult = bigQueryProject.query(bqTableInfo.getSampleQuery());
-    assertThat("Expected number of rows are present", bqQueryResult.getTotalRows(), equalTo(7L));
-
-    List<Map<String, Object>> results =
-        BQTestUtils.mapToList(
-            bqQueryResult, "id", "participant_ids", "date_collected", "derived_from");
-
-    List<Map<String, Object>> dataPostMerge =
-        jsonLoader.loadObjectAsStream(
-            "ingest-test/merge/ingest-test-sample-merge-expected.json", new TypeReference<>() {});
-    List<Map<String, Object>> dataOriginal =
-        jsonLoader.loadObjectAsStream("ingest-test-sample.json", new TypeReference<>() {});
-
-    assertThat(
-        "Only specified records and fields updated by first merge ingest",
-        results,
-        containsInAnyOrder(
-            // Updated rows
-            dataPostMerge.get(0), // ID = sample1
-            dataPostMerge.get(1), // ID = sample2
-            // Untouched rows
-            dataOriginal.get(2), // ID = sample3
-            dataOriginal.get(3), // ID = sample4
-            dataOriginal.get(4), // ID = sample5
-            dataOriginal.get(5), // ID = sample6
-            dataOriginal.get(6) // ID = sample7
-            ));
-
-    // Updating the same row again via merge ingest should succeed
+    assertSampleTableIdColumnRemainsUnchanged(dataset);
+    // Merge ingest request should overwrite value 'sample7' for column 'derived_from'
+    // TODO - This check is failing -- "sample7" is not being overwritten - Is this correct behavior
+    // if the user specifically sets a value to null?
+    //    dataRepoFixtures.assertColumnTextValueCount(
+    //        steward(), datasetId, "sample", "derived_from", "sample7", 0);
```

Comment:
TODO: Check to see if merge functionality is working as expected.
I added this check to make sure there were no longer any "sample7" values returned for the dervived_from column after we ran the merge ingest. However, it's failing b/c it still exists.

Reply:
This is expected. Thanks @okotsopoulos for looking at this. From [Olivia's original PR](https://github.com/DataBiosphere/jade-data-repo/pull/1289):

> Heads up:

> > Callers cannot unset / "null out" fields.

I'll update the test.